docs/installation.rst

```diff
@@ -1,19 +1,9 @@
 Installation
 ============
 
-Catkit2 consists of a core C++ library and a Python package that wraps this library. It also requires a number of C++ third party libraries to facilitate JSON encoding and decoding, Python bindings, linear algebra and high-speed communication over sockets. Compilation requires a C++ compiler conforming to the C++20 standard. Catkit2 provides scripts to automate downloading and installation of its dependencies and main library.
+Catkit2 consists of a core C++ library and a Python package that wraps this library. It also requires a number of C++ third party libraries to facilitate JSON encoding and decoding, Python bindings, linear algebra and high-speed communication over sockets. Compilation requires a C++ compiler conforming to the C++20 standard. Catkit2 performs installation of the core library and Python package using CMake and Conda.
 
-The following will download all third-party C++ dependencies:
-
-.. code-block:: bash
-
-    cd catkit2
-    cd extern
-    ./download.sh
-
-These downloads can be performed on a separate machine with internet connectivity, in case the machine on which the testbed is run does not have an (unfirewalled) internet connection. After downloading, the resulting folders can be copy-pasted in the extern folder on the machine without internet connectivity.
-
-The following will install all downloaded dependencies and create the Conda environment for catkit2.
+The following will install all dependencies and create the Conda environment for catkit2.
```

Comment:
Btw, do you want to add the Py3.7 installation exception that's mentioned in the README here too?